File: 8kyu 1/bonusTime.js
function bonusTime(salary, bonus) {
   return bonus ? '\u00A3' + salary * 10 : '\u00A3' + salary;
}


const bonusTime = (salary, bonus) => `£${salary * (bonus ? 10 : 1)}`;


function bonusTime(salary, bonus) {
   return bonus ? `£${10 * salary}` : `£${salary}`;
}


function bonusTime(salary, bonus) {
   return '£' + salary * (bonus ? 10 : 1);
}


console.log(bonusTime(10000, true), '£100000');
console.log(bonusTime(25000, true), '£250000');
console.log(bonusTime(10000, false), '£10000');
console.log(bonusTime(60000, false), '£60000');
console.log(bonusTime(2, true), '£20');
console.log(bonusTime(78, false), '£78');
console.log(bonusTime(67890, true), '£678900');

File: Python/8py 1/bonus_time.py
"""
|--------------------------------------------------------------------------
| Do I get a bonus?
|--------------------------------------------------------------------------
|
"""

def bonus_time(salary, bonus):
   return f'${salary*10}' if bonus else f'${salary}'


bonus_time = lambda s,b: "${}".format(s*(10*b or 1))


def bonus_time(salary, bonus):
   return "${}".format(salary * (10 if bonus else 1))


def bonus_time(salary, bonus):
   if bonus: salary = salary * 10
   return str.format('${0}', salary)


def bonus_time(salary, bonus):
   return '$' + str(salary * [1,10][bonus])


print(bonus_time(10000, False), '$10000')
print(bonus_time(60000, False), '$60000')
print(bonus_time(2, True), '$20')
print(bonus_time(78, False), '$78')